grails-app/conf/Config.groovy

```diff
@@ -291,14 +291,10 @@ marvl {
 // WFS indexed file info
 indexedFile.fileSizeColumnName = "size"
 
-// Something else is adding an appender (what?!) - get rid of it, else we end up
-// with duplicate logs in production.
-org.apache.log4j.Logger.rootLogger.removeAllAppenders()
```

Comment:
This will actually still result in double logging in prod. I tested it at the time...